spec/index.bs

```diff
@@ -1442,6 +1479,16 @@ shader. Compressed texture formats are provided by extensions. Their naming
 should follow the convention here, with the texture name as a prefix. e.g.
 `etc2-rgba8unorm`.
 
+The <dfn dfn>texel block</dfn> is a single addressable element of an uncompressed texture, or a single
```

Comment:
Same comment about talking about block-based, maybe? @kainino0x @kvark @JusSn what do you think?

Reply:
yeah, if we can just have a separation between block based and texel based, only bringing in the compressed formats in examples, that seems to be cleaner

Reply:
Done